ibis/tests/benchmarks/test_benchmarks.py

```diff
@@ -809,3 +809,29 @@ def test_ibis_duckdb_to_pyarrow(benchmark, sql, ddb) -> None:
 
     expr = con.sql(sql)
     benchmark(expr.to_pyarrow)
+
+
+def test_large_join(benchmark):
```

Comment:
This will be run when pushed to `master`, we don't run the benchmark suite in the `ibis-main.yml` workflow test suite to make sure it's fast.